scripts/build-shotcut.sh

```diff
@@ -44,7 +44,6 @@ LIBOPUS_HEAD=1
 LIBOPUS_REVISION=
 ENABLE_SWH_PLUGINS=1
 FFMPEG_HEAD=0
-FFMPEG_REVISION="origin/release/2.3"
```

Comment:
I would like to have this back so that it matches the others above (FREI0R_REVISION, LIBEPOXY_REVISION, X264_REVISION,...)